### PR TITLE
docs: rewrite `setup_aligned`

### PR DESCRIPTION
**Pull request by MarcosNicolau**
## Description

A rewrite/revision of the `setup_aligned` docs that simplifies the older version and adds a new section to setup telemetry.

### CODE REVIEW

File: docs/3_guides/6.1_explorer.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+This will:
+
+- Start a postgres docker container
+- Run ecto setup
+- Start the explorer on http://localhost:4000.
+
+If you want to run the explorer without docker run:
+
+```shell
+make run_explorer_without_docker
```
Comment (PatStiles) [inline docs/3_guides/6.1_explorer.md]: `make run_explorer_without_docker` -> `make run_explorer_without_docker_db`

File: docs/3_guides/6.1_explorer.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Then you can run:
+
+```bash
+make explorer_fetch_old_batches FROM_BLOCK=<FROM_BLOCK> TO_BLOCK=<TO_BLOCK>
+```
+
+To get operators strategies and restakes data:
+
+```bash
+make explorer_fetch_old_operators_strategies_restakes FROM_BLOCK<FROM_BLOCK>
```
Comment (PatStiles) [inline docs/3_guides/6.1_explorer.md]: `FROM_BLOCK<FROM_BLOCK>` -> `FROM_BLOCK=<FROM_BLOCK>`

**Review by PatStiles — APPROVED** [review]
Left a few nits. Looks good and works

File: docs/3_guides/6_setup_aligned_locally.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+## Batcher
+
+To start the [Batcher](../2_architecture/components/1_batcher.md) locally:
+
+```bash
+make batcher_start_local
+```
+
+This starts a [localstack](https://www.localstack.cloud/) to act as a replacement for S3. If you want to use the batcher under a real `S3` connection you'll need to specify the environment variables under `batcher/aligned-batcher/.env` and then run:
```
Comment (uri-99) [inline docs/3_guides/6_setup_aligned_locally.md]:
```suggestion
This starts a [localstack](https://www.localstack.cloud/) to act as a replacement for S3.

If you want to use the batcher under a real `S3` connection you'll need to specify the environment variables under `batcher/aligned-batcher/.env` and then run:
```

**Review by uri-99 — APPROVED** [review]

File: docs/3_guides/6.1_explorer.md (hunk 1 above)
Comment (MarcosNicolau) [reply docs/3_guides/6.1_explorer.md]: Solved!

File: docs/3_guides/6.1_explorer.md (hunk 2 above)
Comment (MarcosNicolau) [reply docs/3_guides/6.1_explorer.md]: Solved!

**Review by JuArce — APPROVED** [review]

**Pull request merged**